rpc/ub/plot.sh

```diff
@@ -128,19 +128,19 @@ gen_csv() {
     local VAR_OPT="$1"; shift
     local VALUES="$*"
 
-    local NR_MSGS=$(get_val nr_msgs $COMMON_OPTS)
+    local NR_MSGS=$(get_val nr_msgs "$COMMON_OPTS")
```

Comment:
"COMMON_OPTS" contain space separated options? If yes, then you can't put them in double quotes as they will be passed as single argument.

Reply:
Changed as per comment.